fs/fs_zenfs.cc

```diff
@@ -620,12 +620,38 @@ IOStatus ZenFS::ReopenWritableFile(const std::string& fname,
   return OpenWritableFile(fname, file_opts, result, dbg, true);
 }
 
+/* Must hold files_mtx_ */
+IOStatus ZenFS::GetZenFSChildrenNoLock(const std::string& dir,
+                                       bool include_grandchildren,
+                                       std::vector<std::string>* result,
+                                       const IOOptions& options,
+                                       IODebugContext* dbg) {
+  IOStatus s;
+  std::map<std::string, std::shared_ptr<ZoneFile>>::iterator it;
```

Comment:
We can remove this line(L630) and directly use:
`for (auto it = files_.begin(); it != files_.end(); it++) {`